specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/examples/listLinkedWorkspaces.json

```diff
@@ -0,0 +1,36 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "resourceGroup-1234",
+    "workspaceName": "workspace-123",
+    "link-resourceGroup": "linkRg-123"
+  },
+  "responses": {
+    "200": {
+      "body": [
+        {
+          "linkName": "link-12",
+          "linkType": "Synapse",
+          "subscriptionId": "00000000-1111-2222-3333-444444444444",
+          "resourceGroupName": "linkRg-123",
+          "providerNamespace": "Microsoft.Synapse",
+          "resourceType": "workspaces",
+          "workspaceName": "linkedWorkspace-1234",
+          "userAssignedIdentityResourceId": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/resourceGroup-4321/providers/Microsoft.ManagedIdentity/userAssignedIdentities/uai123",
+          "resourceId": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/linkRg-123/providers/Microsoft.Synapse/workspaces/linkedWorkspace-1234"
+        },

```

Comment:
This is not an RPC-compliant resource payload. You should have a top level `id`, `'name`, `type` property and all the resource properties should go into the `properties` property. Please take a look at [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#resource-api-reference](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#resource-api-reference).

Reply:
LinkedWorkspace is not yet a resource.